### PR TITLE
[DEV-1502] Add metadata to webinar page

### PR DESCRIPTION
**Pull request by tommaso1**
#### List of Changes

Add metadata to webinar detail page

#### How Has This Been Tested?

Go to `/webinars/PagoPA-multe`, check the meta tags

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx [hunk 1]
```diff
@@ -13,6 +16,20 @@ export async function generateStaticParams() {
...
+  params,
+}: {
+  params: Params;
+}): Promise<Metadata> {
+  const webinar = await getWebinar(params?.webinarSlug);
+
+  return makeMetadata({
+    title: 'PagoPA',
+    url: `${baseUrl}/webinars/${webinar.slug}`,
+    locale: 'it_IT',
```
Comment (marcobottaro) [inline apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx:29]: The image is missing

**Review by marcobottaro — CHANGES_REQUESTED** [review]

**Review by marcobottaro — DISMISSED** [review]

File: apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx [hunk 2]
```diff
@@ -13,6 +16,21 @@ export async function generateStaticParams() {
...
 
+export async function generateMetadata({
+  params,
+}: {
+  params: Params;
+}): Promise<Metadata> {
+  const webinar = await getWebinar(params?.webinarSlug);
+
+  return makeMetadata({
+    title: 'PagoPA',
```
[the remark this answers is not in the document]
Comment (tommaso1) [reply apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx:27]: It makes sense to me, but the task specifies this very value to be put in the title

[the remark this answers is not in the document]
Comment (kin0992) [reply apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx:27]: @tommaso1 take a look at [this comment on Jira](https://pagopa.atlassian.net/browse/DEV-1502?focusedCommentId=131637)

Then, could you update this PR adding useful values to the metadata?

[the remark this answers is not in the document]
Comment (marcobottaro) [reply apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx:27]: It should be `webinar.title`

**Review by marcobottaro — CHANGES_REQUESTED** [review]
Fix makeMetadata's title

File: apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (marcobottaro) [reply apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx]: @kin0992 I think that this conversation can be resolved now. Other useful metadata values will be managed with Strapi once the webinars are moved to the CMS

**Pull request merged**